tools/mmgotool/commands/root.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var Version = "v0.1-monorepo"
```

Comment:
Should we discard the concept of a version now that we're in lockstep with the monorepo?

Reply:
Makes sense, probably a version is redundant.